whisker-backend/test/cert.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
If possible, I think it would be better to generate these on-demand as part of running the tests that need them (and removing on `make clean`) rather than checking them in. Not strictly required, but I think it's a better model and raises fewer eyebrows when it comes to security reviews of our repository.

Reply:
I didn't actually mean to commit this, I'd rather create these on the fly (with a golang library).